mlir/test/Dialect/Mesh/ops.mlir

```diff
@@ -0,0 +1,69 @@
+// RUN: mlir-opt %s | mlir-opt | FileCheck %s
```

Comment:
There isn't much CHECK in this file?

Reply:
Updated.

I just found my practice about `ops.mlir` test is not good, where only the input IR's validity is checked.